testcases/MachineConfig.py

```diff
@@ -264,3 +265,47 @@ def runTest(self):
             self.cv_HMC.run_command("chsysstate -r lpar -m %s -o on -n %s -f %s" %
                                    (self.system_name, self.lpar_name, self.lpar_prof))
             time.sleep(5)
+
+
+class addOrRemoveIOSlots(MachineConfig):
```

Comment:
please reflect on the latest code base https://github.com/open-power/op-test/pull/771/files

Reply:
we do not need to use parent class, keep this independent class